<!-- Navigation Controller launch file -->

<launch>
	<group ns="MR_NavigationController">
		<node pkg="mr_navigation_controller" name="NavigationController" type="mr_navigation_controller">
			<param name="lineFollowEnableService" value="mrLineFollower/lineUntilQR"/>
			<param name="moveService" value="mrGo/move"/>
			<param name="performAction" value="mrNavigationController/performAction"/>
			<param name="status" value="mrNavigationController/status"/>
			<param name="currentNode" value="mrNavigationController/currentNode"/>
			<param name="searchLimit" value="100"/>
		</node>
	</group>
</launch>

